### PR TITLE
code cleanup

### PR DESCRIPTION
**Pull request by cwaszczuk**

### CODE REVIEW

File: templates/house-commons-members-current.html [hunk 1]
```diff
@@ -58,69 +58,69 @@
...
-    <li class = "result-item">
-        <ul class = "result-item-details">
-          <li class = "result-item-detail-name"> Adams, Nigel </li>
-          <li> MP for Selby & Ainsty </li>
-          <li> Conservative </li>
-          <li><a href="#"> &gt; </a></li>
+    <li class="result-item">
+        <ul class="result-item-details">
+          <li class="result-item-detail-name"> Adams, Nigel</li>
+          <li>MP for Selby & Ainsty</li>
```
Comment (jakemulley) [inline templates/house-commons-members-current.html]: Can we use a HTML entity?

File: templates/house-commons-members-current.html [hunk 2]
```diff
@@ -58,69 +58,69 @@
...
-        <ul class = "result-item-details">
-          <li class = "result-item-detail-name"> Ahmed-Sheikh, Tasmina </li>
-          <li> MP for Ochil and South Perthshire </li>
-          <li> Scottish National Party </li>
+    <li class="result-item">
+        <ul class="result-item-details">
+          <li class="result-item-detail-name"> Ahmed-Sheikh, Tasmina</li>
+          <li>MP for Ochil and South Perthshire</li>
+          <li>Scottish National Party</li>
           <li><a href="#"> &gt; </a></li>
```
Comment (jakemulley) [inline templates/house-commons-members-current.html]: Do you want to remove the extra space?

File: templates/house-commons-members-current.html [hunk 3]
```diff
@@ -58,69 +58,69 @@
...
     </div>
     <div class="ftr">
       <div class="ftr-lgo">
         <div class="ftr-lgo-icn"></div>
       </div>
-      <p> Parliament.uk <a href="#"> digital blog </a></p>
-      <p> Visit the <a href = "#"> current parliament.uk </a> website </p>
-      <p> <a href="#"> Cookie Policy </a></p>
+      <p>Parliament.uk <a href="#">digital blog</a></p>
+      <p>Visit the <a href = "#">current parliament.uk</a> website</p>
```
Comment (jakemulley) [inline templates/house-commons-members-current.html]: Do you want to remove the extra space?

File: templates/house-commons-members-current.html [hunk 4]
```diff
@@ -58,69 +58,69 @@
     <input type="submit" name="select-letter" value="Go">
   </form>
   <!-- Letter Pagination -->
-  <div class = "letter-pagination">
-    <h2 class="current-letter"> A </h2>
-    <a class="next-letter" href = "#"> B &gt; </a>
+  <div class="letter-pagination">
+    <h2 class="current-letter">A</h2>
+    <a class="next-letter" href = "#">B &gt;</a>
```
Comment (jakemulley) [inline templates/house-commons-members-current.html]: Do you want to remove the extra space?

File: templates/people-id.html [hunk 5]
```diff
@@ -76,16 +71,16 @@
...
       </div>
-      <p> Parliament.uk <a href="#"> digital blog </a></p>
-      <p> Visit the <a href ="#"> current parliament.uk </a> website </p>
-      <p> <a href="#"> Cookie Policy </a></p>
+      <p>Parliament.uk <a href="#">digital blog</a></p>
+      <p>Visit the <a href ="#">current parliament.uk</a> website</p>
+      <p><a href="#">Cookie Policy</a></p>
     </div>
   </footer>
   <script src="../_public/javascripts/main.js"></script>
```
Comment (jakemulley) [inline templates/people-id.html]: Do you want to change the static asset URLs to `/javascripts/main.js`?

File: templates/people-members-current.html [hunk 6]
```diff
@@ -19,29 +19,29 @@
...
       </div>
-      <p> Parliament.uk <a href="#"> digital blog </a></p>
-      <p> Visit the <a href ="#"> current parliament.uk </a> website </p>
-      <p> <a href="#"> Cookie Policy </a></p>
+      <p>Parliament.uk <a href="#">digital blog</a></p>
+      <p>Visit the <a href ="#">current parliament.uk</a> website</p>
+      <p><a href="#">Cookie Policy</a></p>
     </div>
   </footer>
   <script src="../_public/javascripts/main.js"></script>
```
Comment (jakemulley) [inline templates/people-members-current.html]: Do you want to change the static asset URLs to /javascripts/main.js?

File: templates/house-commons-members-current.html [hunk 7]
```diff
@@ -58,69 +58,69 @@
...
       </div>
-      <p> Parliament.uk <a href="#"> digital blog </a></p>
-      <p> Visit the <a href = "#"> current parliament.uk </a> website </p>
-      <p> <a href="#"> Cookie Policy </a></p>
+      <p>Parliament.uk <a href="#">digital blog</a></p>
+      <p>Visit the <a href="#">current parliament.uk</a> website</p>
+      <p><a href="#">Cookie Policy</a></p>
     </div>
   </footer>
   <script src="../_public/javascripts/main.js"></script>
```
Comment (jakemulley) [inline templates/house-commons-members-current.html]: Do you want to change the static asset URLs to /javascripts/main.js?

Comment (cwaszczuk) [inline templates/house-commons-members-current.html]: doesn't work, templates are in their own folder and we want to pull in compiled over the raw assets

**Pull request merged**